locust_testplan.py

```diff
@@ -0,0 +1,133 @@
+from locust import HttpLocust, TaskSet, task, events
+from bs4 import BeautifulSoup
+import random
+
+def is_static_file(file):
+    if "/misc" in file:
+        return True
+    elif "/themes" in file:
+        return True
+    else:
+        return False
+
+def random_word():
+    """Return 1 to 12 random characters, a-z inclusive."""
+    import string
```

Comment:
Ok. I have mixed emotions about this, and have read a few blogs/pages about this, but I'm fine with this being our standard.